src/oscilloscope/agilent_common.jl

```
@@ -609,7 +609,7 @@ function verify_time_offset(time_offset)
     return nothing
 end
 
-_convert_to_microseconds(microseconds) = string(ustrip(microseconds)) * "E-06"
+unitful_ms_to_sci_notation(microseconds) = string(ustrip(microseconds)) * "E-06"
```

@gita135 Are you converting millisecond to microsecond or microsecond to second?

2μs = 2e-06 second
2 second = 2e+6μs

I think you may need to convert time in 'μs' to seconds before sending it scope. The scope timebase will then be set in 'μs' because it does not like "μ' in scope command. Please check the datasheet.

---

@deepaksnn Not converting the unit of measurement, but rather the format. The user enters `μs` (via Unitful) but bc the scope doesn't understand that format, we convert it to scientific format for the scope to understand. The param and return value is both microseconds

---

Renamed it to `μs_to_scientific_e_notation()` for clarity

---

Looks good 👍

---

But why make the function so brittle?
What if the user inputs seconds instead of microseconds, then everything will silently fail, right?
As I see it, this version would be robust against the user using different units:
```julia
_convert_to_microseconds(time::Unitful.Time) = string(ustrip(uconvert(µs, time))) * "E-06"
```

Examples:
```julia
julia> using Unitful
julia> using Unitful: s, ms, µs


julia> _convert_to_microseconds(time::Unitful.Time) = string(ustrip(uconvert(µs, time))) * "E-06"
_convert_to_microseconds (generic function with 1 method)

julia> _convert_to_microseconds(1µs)
"1E-06"

julia> _convert_to_microseconds(1ms)
"1000E-06"

julia> _convert_to_microseconds(1s)
"1000000E-06"


julia> _convert_to_microseconds(1)
ERROR: MethodError: no method matching _convert_to_microseconds(::Int64)
Closest candidates are:
  _convert_to_microseconds(::Union{Quantity{T, 𝐓, U}, Level{L, S, Quantity{T, 𝐓, U}} where {L, S}} where {T, U}) at REPL[9]:1
Stacktrace:
 [1] top-level scope
   @ REPL[14]:1

```

---

I agree. This is correct. 
@gita135 Can you fix this and r-test it?